### DISCUSSION TITLE
Can't find testnet on the remix VM environments list

### DISCUSSION THREAD

**Discussion by kobbyeugene79** [Cyfrin/foundry-full-course-cu#2315] [General]
I'm following the Foundry tutorial, and I'm at the point where I need to choose a testnet environment to deploy my SimpleStorage.sol contract to. However, I can't find the "Inject Provider - MetaMask" option. Can someone please point me in the right direction?

**Comment by fqwww** [comment]
you should be able to seeing this in Remix
![image](https://github.com/user-attachments/assets/8c4078f3-6f79-49a6-aa03-d7dcddd7730e)

**Reply by kobbyeugene79** [reply]
It's not there, mine starts from Remix VM (Cancum)

**Reply by kobbyeugene79** [reply]
Oh, because I was using Brave Browser that's why I didn't see it

I switched to Chrome and it's there

**Reply by EngrPips** [reply]
Awesome, Keep crushing it!
